java/integTest/src/test/java/glide/standalone/CommandTests.java

```diff
@@ -266,4 +268,12 @@ public void time() {
                 "Time() result (" + result[0] + ") should be greater than now (" + now + ")");
         assertTrue(Long.parseLong(result[1]) < 1000000);
     }
+
+    @Test
+    @SneakyThrows
+    public void save() {
+        String error = "Background save already in progress";
+        var response = tryCommandWithExpectedError(() -> regularClient.save(), error);
+        assertTrue(response.getValue() != null || response.getKey().equals(OK));
```

Comment:
doesn't it work to try again if it returns an error?

Reply:
it may fail, it may not. even with a sleep